palette/src/lib.rs

```diff
@@ -42,53 +42,49 @@
-//! dimensional vector, and not an *n* dimensional one, where *n* is some
-//! number of possible frequencies of light, is because our eyes contain
-//! only three types of cones. Each of these cones have different sensitivity
-//! curves to different wavelengths of light, giving us three "dimensions"
-//! of sensitivity to color. These cones are often called the S, M, and L
-//! (for small, medium, and large) cones, and their sensitivity curves
-//! *roughly* position them as most sensitive to "red", "green", and "blue"
-//! parts of the spectrum. As such, we can choose only three values to
-//! represent any possible color that a human is able to see. An interesting
-//! consequence of this is that humans can see two different objects which
-//! are emitting *completely different actual light spectra* as the *exact
-//! same perceptual color* so long as those wavelengths, when transformed
-//! by the sensitivity curves of our cones, end up resulting in the same
-//! S, M, and L values sent to our brains.
-//!
-//! A **color space** (which simply refers to a set of standards by which
-//! we map a set of arbitrary values to real-world colors) which uses
-//! tristimulus values is often defined in terms of
-//!
-//! 1. Its **primaries**
-//! 2. Its **reference white** or **white point**
-//!
-//! The **primaries** together represent the total *gamut* (i.e. displayable
-//! range of colors) of that color space, while the **white point** defines
-//! which concrete tristimulus value corresponds to a real, physical white
-//! reflecting object being lit by a known light source and observed by the
-//! 'standard observer' (i.e. a standardized model of human color perception).
-//!
-//! The informal "RGB" color space is such a tristimulus color space, since
-//! it is defined by three values, but it is underspecified since we don't
-//! know which primaries are being used (i.e. how exactly are the canonical
-//! "red", "green", and "blue" defined?), nor its white point. In most cases,
-//! when people talk about "RGB" or "Linear RGB" colors, what they are
-//! *actually* talking about is the "Linear sRGB" color space, which uses the
-//! primaries and white point defined in the sRGB standard, but which *does
-//! not* have the (non-linear) sRGB *transfer function* applied.
-//!
-//! This library takes these things into account, and attempts to provide an
-//! interface which will let those who don't care so much about the intricacies
-//! of color still use colors correctly, while also allowing the advanced user
-//! a high degree of flexibility in how they use it.
+//! See the [rgb] module for a deeper dive into RGB and (non-)linearity.
+//!
+//! # Color Spaces
+//!
+//! "RGB" and other tristimulus based spaces like CIE Xyz are probably the most
+//! widely known color spaces. These spaces are great when you want to perform
+//! physically correct math on color (like in a 2d or 3d rendering program) but
```

Comment:
Hmm, "physically based" is probably a better term. I'm not 100% sure what the exact intention was (I didn't write this part, myself), but I believe it will get the same point across.